src/containers/StructureVersion/TaxonomyVersionProvider.tsx

```diff
@@ -27,11 +27,21 @@ export const TaxonomyVersionProvider = ({ children }: Props) => {
 
 interface TaxonomyVersion {
   taxonomyVersion: string;
+  changeVersion: (newHash: string) => void;
 }
 
 export const useTaxonomyVersion = (): TaxonomyVersion => {
   const versionContext = useContext(TaxonomyVersionContext);
+  const changeVersion = (newHash: string) => {
+    if (!versionContext) {
+      throw new Error(
+        'You cannot change the taxonomy version without having a TaxonomyVersionProvider present!',
```

Comment:
i18n? Eller gjør vi ikkje det for slike feilmeldinger?

Reply:
Vi har ikke pleid å gjøre det, da disse feilmeldingene kun kommer opp under utvikling.